lib/skate_web/router.ex

```diff
@@ -112,7 +131,8 @@ defmodule SkateWeb.Router do
       :accepts_html,
       :browser,
       :auth,
-      :ensure_auth
+      :ensure_auth,
+      :ensure_environment_access
```

Comment:
thought(non-blocking): I'm thinking of opening a ticket for this(if there isn't one), but I think maybe we should consider adding trailing comma's to our formatter config? It'd be nice to take off the mental overhead of having to realize that the change that occurred to `:ensure_auth` was adding a `,`. (Since github isn't highlighting the only character that changed on line `134`)

Thoughts @mbta/skate-developers ?

Reply:
I like that idea in general, but I don't believe it's actually configurable - the Elixir code formatter explicitly lists "provide as little configuration as possible" as one of its [design principles](https://hexdocs.pm/elixir/main/Code.html#format_string!/2-design-principles) and I don't see an option for trailing commas.